app/views/govuk_component/breadcrumbs.raw.html.erb

```diff
@@ -1,18 +1,20 @@
 <%
   breadcrumbs ||= []
   collapse_on_mobile ||= false
+  inverse ||= false
+  collapse_class =  collapse_on_mobile && breadcrumbs.any? { |crumb| crumb[:is_page_parent] } ? "collapse-on-mobile" : ""
+  invert_class = inverse ? "govuk-breadcrumbs--inverse" : ""
 %>
 
-<div
-  class="govuk-breadcrumbs <%= "collapse-on-mobile" if collapse_on_mobile && breadcrumbs.any? { |crumb| crumb[:is_page_parent] } %>"
-  data-module="track-click">
+<div class="govuk-breadcrumbs <%= invert_class %> <%= collapse_class %>" data-module="track-click">
   <ol>
   <% breadcrumbs.each_with_index do |crumb, index| %>
     <%
       is_link = crumb[:url].present? || crumb[:is_current_page]
       path = crumb[:is_current_page] ? '#content' : crumb[:url]
       aria_current = crumb[:is_current_page] ? 'page' : 'false'
       css_class = crumb[:is_current_page] ? 'breadcrumb-for-current-page' : ''
+      css_class << 'breadcrumb-for-current-page--inverse' if inverse && crumb[:is_current_page]
```

Comment:
This currently results in an element getting the 2 classes without any space between them, e.g:

`breadcrumb-for-current-pagebreadcrumb-for-current-page--inverse`

Reply:
I think we can simplify this to get `govuk_breadcrumbs--inverse` to apply to links and current links. We could:

- Continue to construct the inverse class at the top
- Instead of applying it to the overall govuk-breadcrumbs div, apply it to each link
- This would replace the last line with:

`css_class << invert_class`

We would then need to add the following to the CSS, under the current `breadcrumb-for-current-page` rule

```
  .govuk-breadcrumbs--inverse,
  .govuk-breadcrumbs--inverse a {
    color: $white;
  }
```

Reply:
See the latest commit - I've implemented this, but had to be more specific with the css to override what we're pulling in previously, which is starting to get a bit ugly - seemed better than throwing !important all over it, though. Thoughts? If you're happy with that css I'll squash the commit.